typescript/sdk/src/zksync/ZKSyncDeployer.ts

```diff
@@ -0,0 +1,198 @@
+import assert from 'assert';
```

Comment:
wrong assert. I thought we had a lint rule to catch these but maybe it stopped working

Reply:
Will fix. Its also being used here btw. [EvmIsmModule.hardhat-test.ts](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/db5875cc22747362cd3e0a275b4897f1c3b7a800/typescript/sdk/src/ism/EvmIsmModule.hardhat-test.ts#L2).